std/testing/asserts.ts

```diff
@@ -133,12 +128,10 @@ export function equal(c: unknown, d: unknown): boolean {
   })(c, d);
 }
 
-/** Make an assertion, if not `true`, then throw. */
-export function assert(expr: unknown, msg = ""): asserts expr {
-  if (!expr) {
-    throw new AssertionError(msg);
-  }
-}
+export const assert: (
+  condition: unknown,
+  message?: string | undefined
+) => asserts condition = Deno.assert;
```

Comment:
Unconditionally assigning `Deno.assert` makes the asserts module incompatible with other runtimes.
Maybe it should be done like in line 18 to preserve compatibility?